File: api/Floricultura/Floricultura.Domain/Models/Menu/Menu.cs
namespace Floricultura.Domain.Models.Menu
{
    public class Menu
    {
        public int Id { get; set; }
        public string? Item { get; set; }
        public string? Rota { get; set; }
        public int Ordem { get; set; }
        public IEnumerable<SubMenu>? SubMenus { get; set; }
    }
}

File: site/src/app/models/Menu.ts
export interface Menu {
  id: number;
  item: string;
  rota: string;
  ordem: number;
  subMenus: Array<SubMenu>;
}
export interface SubMenu {
  id: number;
  item: string;
  rota: string;
  ordem: number;
  idMenu: number;
}
